sui/its.js

```diff
@@ -997,6 +1061,14 @@ if (require.main === module) {
             },
         );
 
+    const mintCoinsProgram = new Command()
+        .name('mint-coins')
+        .command('mint-coins <coinPackageId> <coinPackageName> <coinModName> <amount> <receiver>')
```

Comment:
Let's use `<symbol>` to load the coin from the user's config, so we don't need to collection coin package id, coin package name, coin module name etc. See `link-coin` etc. for example. We can still keep them as options to support coins not in your config.

Let's change `receiver` to `recipient` to be consistent with the language in `its-example.js`